src/Bubble.js

```diff
@@ -60,8 +60,8 @@ export default class Bubble extends React.Component {
     if (this.props.renderTicks) {
         return this.props.renderTicks(currentMessage);
     }
-    if (currentMessage.user._id !== this.props.user._id) {
-        return;
+    if (isFromOtherUser({ currentMessage, user: this.props.user })) {
```

Comment:
I see that you're using property/value shorthands for the `currentMessage` taking advantage of ES6's Enhanced Object Literals. IMO, is better to keep coherence. I mean, if you cannot use the shorthands for all properties, is preferable to avoid it since it's a little confusing.

Furthermore, for this kind of functions is more clear to pass it the arguments to compare separately instead of inside an object. However, I think that at this point we should to ping @kfiroo to get his opinion and recommendations.